bootstrap/_rpm_common.sh

```diff
@@ -5,15 +5,29 @@
 #   - Centos 7 (x64: on AWS EC2 t2.micro, DigitalOcean droplet)
 
 # "git-core" seems to be an alias for "git" in CentOS 7 (yum search fails)
-yum install -y \
-  git-core \
-  python \
-  python-devel \
-  python-virtualenv \
-  python-devel \
-  gcc \
-  dialog \
-  augeas-libs \
-  openssl-devel \
-  libffi-devel \
-  ca-certificates \
+bootstrap() {
```

Comment:
Why are we wrapping this into a function? Seems not in line with `_deb_common.sh`